README.md

```diff
@@ -60,6 +60,7 @@ If you are using [lazy.nvim](https://github.com/folke/lazy.nvim). Add the follow
 ```lua
 {
   "christoomey/vim-tmux-navigator",
+  event = "Very Lazy",
```

Comment:
```suggestion
  event = "VeryLazy",
```
The event has to be called 'VeryLazy' and it just seems like a careless mistake to me.

Notes: I just tried this change and it fixed the local problem. Thanks.

Reply:
Yikes you’re right, I even got it right in the description. 

I still wonder if bufpreread isn’t better in some way as that’s what I’ve seen more often. VeryLazy just appears in the lazyvim docs.